src/test/components/__snapshots__/Details.test.js.snap

```diff
@@ -51,7 +51,7 @@ exports[`app/Details renders an initial view with the right panel 1`] = `
   <ProfileCallTreeView />
```

Comment:
So few changes here for so big changes in the code... our coverage is not great in this area.

Do you think you could add some component testing around these new components ?

Reply:
Yes, I think this is the first major code change to this section. I will write out my testing plan to the other PR, as so far it's just been in my head. I would like to do it as a follow-up to landing this code though.